app/graphql/types/account.rb

```diff
@@ -15,6 +15,62 @@ class Types::Account < Types::BaseObject
     null: true,
     description: 'The PRO subscription for this account'
 
+  # field :pro_started_at, GraphQL::Types::ISO8601DateTime,
+  #   null: true,
+  #   description: 'Time PRO subscription started for the account'
+  #
+  # # unsure if this is the right place
+  # field :pro_expires_at, GraphQL::Types::ISO8601DateTime,
+  #   null: true,
+  #   description: 'Time PRO subscription expires for the account'
```

Comment:
Just remove these for now, they'll be on the ProSubscription when I get back to work on Pro